docs/run-platform-server/ingestion.mdx

```diff
@@ -34,29 +43,15 @@ stellar-horizon db reingest range <start> <end>
 
 </CodeExample>
 
-_(The command name is a bit of a misnomer: you can use `reingest` both to ingest new ledger data and reingest old data.)_
-
-You can run this process in the background while your Horizon server is up. It will continuously decrement the `history.elder_ledger` in your `/metrics` endpoint until the `<end>` ledger is reached and the backfill is complete. If Horizon receives a request for a ledger it hasn't ingested, it returns a 503 error and clarify that it's `Still Ingesting` (see [below](#some-endpoints-are-not-available-during-state-ingestion)).
-
-#### Deciding on how much history to ingest
+Running any historical range of ingestion requires coordination with the data retention configuration chosen. If you have a temporal limit on history set with `HISTORY_RETENTION_COUNT=<LEDGER_COUNT>` then it makes no sense to ingest any time range that is older as it will get purged from the database almose as soon as it's added.
 
-You should think carefully about the amount of ingested data you'd like to keep around. Though the storage requirements for the entire Stellar network are substantial, **most organizations and operators only need a small fraction of the history** to fit their use case. For example,
+Typically the only time you need to run historical ingestion is once when boot-strapping a system after first deployment, from that point forward **live** ingestion will keep the database populated with the expected sliding window of trailing historical data. Maybe one exception is if you think you have a gap in the database caused by the **live** ingestion being down, in which case you can run historical ingestion range to essentially gap fill.
 
-- If you just started developing a new application or service, you can probably get away with just doing live ingestion, since nothing you do requires historical data.
+You can run historical ingestion in parallel in background while your main Horizon server separately performs **live** ingestion. If the range specified overlaps with data already in the database, it will simply be overwritten, effectively idempotent.
```

Comment:
It might be helpful to explicitly mention that overlapping range is not an issue with something like "it is OK to specify a range that overlaps with existing data in the database...."